lib/ParamedicPermissions.js

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+
+/* jshint node: true */
+var path 	 = require('path');
+var fs 		 = require('fs');
+var shelljs  = require("shelljs");
+var logger  = require('./utils').logger;
+
+function ParamedicPermissions(appName, simulatorsFolder, dbPath) {
+	this.appName = appName;
+	this.simulatorsFolder = simulatorsFolder;
+	this.dbPath = dbPath;
+}
+
+ParamedicPermissions.prototype.updatePermissions = function(serviceList){
+	var self = this;
+	this.getDirectories(self.simulatorsFolder).forEach(function(simFolder){
+		var destinationTCCFile = path.join(self.simulatorsFolder, simFolder, '/data/Library/TCC/TCC.db');
+
+		//Check if the simFolder has TCC.db file in it
+		if(self.doesFileExist(destinationTCCFile)) {
+			var sqlite3Command_firstPart  = 'sqlite3 ' + destinationTCCFile;
+			var sqlite3Command_secondPart = ' "insert into access(service, client, client_type, allowed, prompt_count, csreq) values(\'';
```

Comment:
Instead of copying a tcc db with permissions - consider copying empty and insert permissions into it - that helps in configuring permissions only in one place.